extra-tools/index.md

```diff
@@ -60,7 +63,9 @@ The `groupName` filter makes it possible for you to define aliases for your grou
 }
 {% endhighlight %}
 
-Note that this filter rewrites the `group` key from each item. For instance, an item with such a group key:
+<p class="note  note--info"><strong>Note:</strong>  The `groupName` filter overrides the `group` key of each item.</p>
+
+Result of using the `group` key:
```

Comment:
It's not the result of a filter, just the default data from SassDoc. 
This would need to be rewritten into something like "Dafault `group` key".